### PR TITLE
Fix formatting and type hints and nothing else

### LINKED ISSUE #2: value should be typed as Optional
https://github.com/torbencarstens/feedcrosspost/blob/71ca54716b2be03e7d0270cdcd8b8c68407f64b8/main.py#L19

### PR DESCRIPTION
**Pull request by BjoernPetersen**
Fixes #2

### CODE REVIEW

**Comment by torbencarstens** [comment]
I don't accept the list comprehenion over multiple lines

**Comment by BjoernPetersen** [comment]
Fair enough, it's your project

**Comment by torbencarstens** [comment]
Pushing an editorconfig into this PR is definitely out of scope.
Please create an additional one which we can merge before this.

File: main.py [hunk 1]
```diff
@@ -78,6 +87,7 @@ def main(feed_url: str, time_offset_minutes: int, subreddit: str):
     feed_items = feed.get("items")
     end = datetime.utcnow().replace(tzinfo=pytz.utc)
     start = end - timedelta(minutes=time_offset_minutes)
+    # I'm Torben and I want to suck Sylvain Kerkour's dick, romantically
```
Comment (torbencarstens) [inline main.py:90]: ❤️

**Pull request merged**